src/bio/bio_config.c

```diff
@@ -773,66 +773,47 @@ decode_daos_data(const char *nvme_conf, const char *method_name, struct config_e
 	/* Get 'config' array first configuration entry */
 	ctx->config_it = spdk_json_array_first(ctx->config);
 	if (ctx->config_it == NULL) {
-		/* Entry not-found so return positive RC */
-		D_GOTO(out, rc = 1);
+		D_DEBUG(DB_MGMT, "%s entry not found in spdk bootstrap config\n", method_name);
+		D_GOTO(out, rc = JSON_NOT_FOUND);
```

Comment:
Should it not be better to convert it to a DAOS error such as `DER_NONEXIST` ?

Reply:
keeping as is so we have the same change on master and 2.6 branches